### DISCUSSION TITLE
RAM cache for NVMe drives

### DISCUSSION THREAD

**Discussion by adamsky-01** [madMAx43v3r/chia-plotter#527] [Ideas]
Could it be beneficial to create the RAM cache for NVMe drive and use it for tmpdir2 with madMAx43v3r plotter? I'm waiting for 128GiB memory to make the RAM disk but this a different animal. 
I've 48 GiB RAM in each of my two plotting machines. Most of this memory is not used when plotting with madMAx43v3r plotter. I can reserve approx. 35 GiB of RAM for NVMe drive cache. Could it reduce the NVMe SSD usage and prolong its life or even slightly decrease the plotting time or it's not really useful in such application?

**Comment by 85256638** [comment]
would like to know too, what software could be used to set a portion of RAM as the cache for NVME/SSD? I found this PrimoCache software seems to do the work, but I have never tested.
